images/router/haproxy/conf/haproxy-config.template

```diff
@@ -32,6 +35,10 @@ global
 defaults
   # maxconn 4096
   # Add x-forwarded-for header.
+{{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
+  option httplog
```

Comment:
Does this have any ill effect on passthrough routes which are tcp typed (others have `mode http` in their template definitions)?